### PR TITLE
feat: new log design and capture area tooltip

### PR DESCRIPTION
**Pull request by BeierKevin**
# PR description
*Describe your changes in detail here*

Proper collapsable items for the capture area logs and tooltip for the capture area rectangles.

# Definition Of Done (DoD)
*This PR can be squashed / merged if*
- [x] a developer is assigned
- [x] the PR is NOT estimated
- [x] the PR is labeled
- [x] the PR is assigned to the current sprint
- [x] a meaningful title has been set according to https://www.conventionalcommits.org/
- [x] the PR is described in detail
- [x] the PR links to an issue
- [x] the PR has been reviewed

*Add additional conditions here if necessary for this PR*

fiex: #69

### CODE REVIEW

File: src/components/LogOutput.vue [hunk 1]
```diff
@@ -1,24 +1,35 @@
...
-                    <v-icon color="primary" icon="mdi-information"></v-icon>
-                </template>
+    <v-expansion-panel class="log-output-container">
+        <v-expansion-panel-title class="pa-4" expand-icon="mdi-menu-down">
+            Capture area
+            {{ captureAreaId }}
+        </v-expansion-panel-title>
+        <v-expansion-panel-text>
+            <p v-if="matchedElementsIsEmpty">
+                Start the capturing to get some values!
```
Comment (Claiyc) [inline src/components/LogOutput.vue:9]:
```suggestion
                Start capturing first!
```

File: src/views/RunPage.vue [hunk 2]
```diff
@@ -19,11 +19,17 @@
...
         <template v-slot:default>
-            <LogOutput
-                v-for="captureArea in captureAreas"
-                :key="captureArea.getId()"
-                :captureAreaId="captureArea.getId()"
-            />
+            <v-alert v-if="isRunning" class="mb-4" type="warning"
+                >While capturing you are unable to switch changes sources or
+                adjust regex, please stop if you want to change those!</v-alert
+            >
```
Comment (Claiyc) [inline src/views/RunPage.vue:25]:
```suggestion
```
Unnecessary imo. I know you want to use these alert boxes, we'll find use for them at some point :D

**Review by Claiyc — CHANGES_REQUESTED** [review]
Can we make it so that the first capture area in the run view is always open by default? Would be a nice feature imo

File: src/views/RunPage.vue (hunk 2 above)
Comment (BeierKevin) [reply src/views/RunPage.vue:25]: But the navigation is blocked as information to the user, but Ok 🌵

**Review by Claiyc — APPROVED** [review]
Beautiful

**Pull request merged**